pkg/investigations/clusteroperatordownhcp/clusteroperatordownhcp_test.go

```
@@ -0,0 +1,230 @@
+
+import (
+	"errors"
+	"testing"
+
+	ec2v2types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+
+	"github.com/openshift/configuration-anomaly-detection/pkg/executor"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)